scripts/ingests/ingest_utils.py

```diff
@@ -0,0 +1,782 @@
+                                                                          source_pm_data['mu_dec_error']))). \
+                        values(adopted=True)
+                    db.engine.execute(adopted_pm)
+            logger.debug("!!! Another Proper motion exists")
+            if logger.level == 10:
+                source_pm_data.pprint_all()
+
+        else:
+            msg = 'Unexpected state'
+            logger.error(msg)
+            raise RuntimeError(msg)
+
+        # Construct data to be added
+        pm_data = [{'source': db_name,
+                    'mu_ra': pm_ras[i],
+                    'mu_ra_error': pm_ra_errs[i],
+                    'mu_dec': pm_decs[i],
+                    'mu_dec_error': pm_dec_errs[i],
+                    'adopted': adopted,
+                    'reference': pm_references[i]}]
+        logger.debug(f'Proper motion data to add: {pm_data}')
+
+        try:
+            db.ProperMotions.insert().execute(pm_data)
+            n_added += 1
+        except sqlalchemy.exc.IntegrityError:
+            msg = "The source may not exist in Sources table.\n" \
+                  "The proper motion reference may not exist in Publications table. " \
+                  "Add it with add_publication function. \n" \
+                  "The proper motion measurement may be a duplicate."
+            logger.error(msg)
+            raise SimpleError(msg)
+
+        updated_source_pm_data = db.query(db.ProperMotions).filter(db.ProperMotions.c.source == db_name).table()
+        logger.info('Updated proper motion data:')
+        if logger.level == 20: # Info = 20, Debug = 10
+            updated_source_pm_data.pprint_all()
+
+    return
+
+
+# PHOTOMETRY
+def ingest_photometry(db, sources, bands, magnitudes, magnitude_errors, reference, ucds=None,
+                      telescope=None, instrument=None, epoch=None, comments=None):
+    """
+    TODO: Write Docstring
+
+    Parameters
+    ----------
+    db
+    sources
+    bands
+    magnitudes
+    magnitude_errors
+    reference
+    ucds
+    telescope
+    instrument
+    epoch
+    comments
+
+    Returns
+    -------
+
+    """
+
+    n_added = 0
+
+    n_sources = len(sources)
+
+    if n_sources != len(magnitudes) or n_sources != len(magnitude_errors):
+        msg = f"N Sources:, {len(sources)}," \
+              f" N Magnitudes, {len(magnitudes)}, N Mag errors:, {len(magnitude_errors)}," \
+              f"\nSources, magnitudes, and magnitude error lists should all be same length"
+        logger.error(msg)
+        raise RuntimeError(msg)
+
+    if isinstance(bands, str):
+        bands = [bands] * len(sources)
+
+    if isinstance(reference, str):
+        reference = [reference] * len(sources)
+
+    if isinstance(telescope, str):
+        telescope = [telescope] * len(sources)
+
+    if isinstance(instrument, str):
+        instrument = [instrument] * len(sources)
+
+    if isinstance(ucds, str):
+        ucds = [ucds] * len(sources)
```

Comment:
if you like, the following also works
```suggestion
things = [bands, reference, telescope, instrument, ucds]
for i, thing in enumerate(things):
    if isinstance(thing, str):
        thing = [thing] * len(sources)
        things[i] = thing
bands, reference, telescope, instrument, ucds = things
```